app/src/main/kotlin/com/wire/android/ui/WireActivity.kt

```diff
@@ -525,6 +531,14 @@ class WireActivity : AppCompatActivity() {
         }
     }
 
+    private fun handleSynchronizeExternalData(intent: Intent) {
+        intent.data?.lastPathSegment.let { eventsPath ->
```

Comment:
Maybe limit this to debug builds and close this door on production?

`if (BuildConfig.DEBUG) {`

Reply:
Done